bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -213,6 +222,18 @@ public Builder eventLoopGroup(EventLoopGroup f) {
             return this;
         }
 
+        /**
+         * Configure the bookkeeper client with a provided {@link ByteBufAllocator}.
+         *
+         * @param allocator an external {@link ByteBufAllocator} to use by the bookkeeper client.
+         * @return client builder.
+         * @since 4.9
+         */
+        public Builder eventLoopGroup(ByteBufAllocator allocator) {
```

Comment:
allocator?

Reply:
Can you also add the method to BookKeeperBuilder? 

https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/BookKeeperBuilder.java